src/backend/database/migrations/micropowermanager/2025_01_04_120250_drop_meter_parameter_id_field_on_social_tariff_piggy_banks_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
```

Comment:
We can also remove, all things related to SocialPiggyBank. It was a custom feature for one of MPM instances (in Tanzania). There is no any tenant company using this feature no longer.

Reply:
Ok, good to know. Let's do that in a separate PR.

Reply:
Created issue for this via  #531